consensus/istanbul/uptime/uptime.go

```diff
@@ -173,11 +166,8 @@ func (um *Monitor) ProcessBlock(block *types.Block) error {
 	return nil
 }
 
-// updateUptime receives the currently accumulated uptime for all validators in an epoch and proceeds to update it
-// based on which validators signed on the provided block by inspecting the block's parent bitmap
-//
-// It expects a blockNumber with the signatures bitmap from the previous block (parent seal)
-func updateUptime(uptime *Uptime, blockNumber uint64, bitmap *big.Int, window uint64, epochNum uint64, epochSize uint64) *Uptime {
+// updateUptime update accumulated uptime given a block and its validator's signatures bitmap
+func updateUptime(uptime *Uptime, blockNumber uint64, bitmap *big.Int, lookbackWindowSize uint64, monitoredWindow Window) *Uptime {
```

Comment:
Maybe rename the argument to `monitoringWindow` to be consistent?  I think `monitored` is fine when it comes to blocks within the window (as below)